kvgraph/graph.go

```diff
@@ -360,12 +360,12 @@ func (kgdb *KVInterfaceGDB) GetOutChannel(reqChan chan gdbi.ElementLookup, load
 				if err == nil {
 					_, gid := VertexKeyParse(req.data)
 					v := &gripql.Vertex{Gid: gid}
-					if load {
```

Comment:
During grids development, uncovered a bug. This is fixing a special case when vertex label values wouldn't be loaded when needed. (because their stored along side the rest of the vertex data)